netmanager-app/components/grids/polymap.tsx

```
@@ -0,0 +1,120 @@
+interface DrawEvent {
+  layerType: string;
+  layer: L.Layer;
+}
+
+// Type for edit events
+interface EditEvent {
+  layers: L.LayerGroup;
+}
+
+// Fix Leaflet's default icon issue
+delete (L.Icon.Default.prototype as IconDefault)._getIconUrl;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)